library/x509.c

```diff
@@ -131,6 +131,48 @@ int mbedtls_x509_get_alg( unsigned char **p, const unsigned char *end,
     return( 0 );
 }
 
+/*
+ * Convert md type to string
+ */
+static inline const char* md_type_to_string( mbedtls_md_type_t md_alg )
```

Comment:
Note: I was going to complain that large inline functions are bad for code size, but in this instance since the function is only declared in single `.c` file, that's not a problem: with `-Os` the compiler will ignore the `inline` hint so this file will contain only on copy, and no other file will contain a copy. (No change requested, I just wanted to record my though process in case the other reviewer wonders about the same thing.)